src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const bookSlice = createSlice({
+  name: 'books',
+  inialState: [],
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    deleteBook: (state, action) => {
+      state.books.filter((book) => book.id !== action.payload.id);
+    },
+  },
+});
+
+export const { addBook, deleteBook } = bookSlice.actions;
+
+export default bookSlice.reducer;
```

Comment:
When I  run npm start your page is opening up but its blank and not displaying any content. And its due to the following error in the console. Please take a look at the screen shot below:

Screen shot displaying error message from the console:

![Screenshot 2023-04-19 at 1 59 30 PM](https://user-images.githubusercontent.com/79506105/233069581-5686d01e-be81-44da-a6eb-687f3ac25abf.png)

Your page isnt displaying any content and its due to that error above please it would be nice if you fixed this issue and please always  make sure you test and check if everrything is functioning properly before you submit your work so you dont finish your review submission chances.